### PR TITLE
Provide "Add new cluster" as welcome page

### PR DESCRIPTION
**Pull request by angelozerr**
Provide "Add new cluster" as welcome page

Signed-off-by: azerr <azerr@redhat.com>

### CODE REVIEW

**Comment by angelozerr** [comment]
This PR provides the standard vscode mechanism ( welcome page) to provide  a button to ad new cluster when treeview is empty:

image

**Comment by angelozerr** [comment]
For the CHANGELOG, I don't know if I must update something, since it replaces the no cluster item PR which belongs to this millestone.

File: package.json [hunk 1]
```diff
@@ -350,6 +350,12 @@
                 }
             ]
         },
+        "viewsWelcome": [
+            {
+                "view": "kafkaExplorer",
+                "contents": "No clusters found.\n[Add new cluster](command:vscode-kafka.explorer.addcluster)"
```
Comment (fbricon) [inline package.json:356]: No Kafka clusters found

**Comment by fbricon** [comment]
> For the CHANGELOG, I don't know if I must update something, since it replaces the no cluster item PR which belongs to this millestone.

change link to #76 (or add) with this PR

File: package.json (hunk 1 above)
Comment (angelozerr) [reply package.json]: fixed

**Comment by angelozerr** [comment]
> change link to #76 (or add) with this PR

updated.

**Comment by fbricon** [comment]
@angelozerr thanks, looks way better and removes some code at the same time. My favorite kind of change :-)

**Pull request merged**